test/app-luatest/fix_lua_loader_error_handling_test.lua

```diff
@@ -14,22 +14,16 @@ local g = t.group()
 -- the end of the test.
 local cwd = fio.cwd()
 
-local luadir
 local name = 'syntax'
-
--- XXX: Unfortunately, multiline literals do not interpolate
--- escape sequences, so just concatenate two lines of the error
--- message to build the final template.
-local errlike = table.concat({
-    "error loading module '<NAME>' from file '<DIR>/<NAME>.lua':",
-    "<DIR>/<NAME>.lua:1: unexpected symbol near '?'",
-}, "\n\t")
-
+-- XXX: luatest uses only `string.find()` with 4th argument
+-- (`plain`) unconditionally set to `true` for error matching, so
+-- there is no way to use any pattern here.
+local errlike = ("/%s.lua:1: unexpected symbol near '?'"):format(name)
```

Comment:
Since it's not pattern or format string anymore, so the variable is `errmsg`, isn't?

Reply:
Replaced with `errtail`. Branch is force-pushed, see the iterative patch below.
```diff
diff --git a/test/app-luatest/fix_lua_loader_error_handling_test.lua b/test/app-luatest/fix_lua_loader_error_handling_test.lua
index 45d188616..6cf251121 100644
--- a/test/app-luatest/fix_lua_loader_error_handling_test.lua
+++ b/test/app-luatest/fix_lua_loader_error_handling_test.lua
@@ -18,7 +18,7 @@ local name = 'syntax'
 -- XXX: luatest uses only `string.find()` with 4th argument
 -- (`plain`) unconditionally set to `true` for error matching, so
 -- there is no way to use any pattern here.
-local errlike = ("/%s.lua:1: unexpected symbol near '?'"):format(name)
+local errtail = ("/%s.lua:1: unexpected symbol near '?'"):format(name)
 
 g.before_all(function()
     treegen.init(g)
@@ -33,5 +33,5 @@ g.after_all(function(g)
 end)
 
 g.test_loader_error_handling = function()
-    t.assert_error_msg_contains(errlike, require, name)
+    t.assert_error_msg_contains(errtail, require, name)
 end
```